readme.md

```diff
@@ -79,28 +134,341 @@ The default export is `rehypeSanitize`.
 
 ### `unified().use(rehypeSanitize[, schema])`
 
-Remove potentially dangerous things from HTML, or more correct: keep only the
-safe things in a document.
+Sanitize HTML.
 
 ###### `schema`
 
-The sanitation schema defines how and if nodes and properties should be cleaned.
-The schema is documented in [`hast-util-sanitize`][schema].
+Sanitation schema that defines if and how nodes and properties should be
+cleaned.
 The default schema is exported as `defaultSchema`.
 
+This option is a bit advanced as it requires knowledge of ASTs, so we defer
+to the documentation available for [`Schema` in `hast-util-sanitize`][schema].
+
+## Example
+
+### Example: headings (DOM clobbering)
```

Comment:
interesting and elaborate example

Reply:
Apparently GH has source maps on their site so it was possible to reverse engineer how they exactly do it. Took some time to remove all the stuff specific to the libraries they use